happi/cli.py

```diff
@@ -0,0 +1,110 @@
+"""
+Module for a command line interface to happi
+"""
+# cli.py
+# NOTE: Does not work from any environment, must have pcds_conda activated
+## Can this script activate/deactivate pcds_conda upon use?
+
+import argparse
+import sys
+import happi.backends.json_db as json_db
+import happi.client as hcl
+
+parser = argparse.ArgumentParser(description='happi command line tool')
+
+# First add search argument
+parser.add_argument('--search', action='store_true',
+                    help='search for device')
+parser.add_argument('--field', help='field in happi database to consider')
+parser.add_argument('--value', help="value of 'field' to return results",
+                    type=str)
+
+parser.add_argument('--add', action='store_true',
+                    help='add a device')
+
+# TODO: Implement edit option
+#parser.add_argument('--edit', action='store_true',
+#                    help='edit a device')
+
+#args = parser.parse_args()
+#arg_dict = vars(args)
+
+def happi_cli(args):
+    args = parser.parse_args(args)
+    print(args)
```

Comment:
Use the `logger` here. That way you don't have to remember to delete them. Just give it a little nicer message